addons/mail/static/src/models/message_list_view/message_list_view.js

```diff
@@ -0,0 +1,20 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one2one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'MessageListView',
+    identifyingFields: ['threadView'],
```

Comment:
Maybe have Owner in the name for consistency with others?

I think it helps a lot in general to always have Owner in identifying fields name, so that in the rest of the code it's clear when referencing a related component record if it's an owner or a children.